Source/SourceGenerators/Blazorise.Generator.Features/Blazorise.Generator.Features.csproj

```diff
@@ -16,6 +16,8 @@
     <Copyright>Copyright 2018-2024 Megabit</Copyright>
 
     <TargetFramework>netstandard2.0</TargetFramework>
+
+    <LangVersion>latest</LangVersion>
```

Comment:
I noticed that you added `<LangVersion>latest</LangVersion>` in a few places. Do we need it?

Reply:
I agree it is convenient. But remember, we support net6 - net9, and for older frameworks, newer C# is not available.

Reply:
For some features it's not, but for most of them it is (for example the collections)...  The support is assured by `<TargetFraeworks>` not by version of c#... 
Well - it would not compile, right? So having the c# versions set to `latest` together with target frameworks should be safe... I can use file-scoped namespace and collection expression inside `Weavers.Fody` and it happily compiles for `netstandard2.0`...